rollup.config.js

```diff
@@ -0,0 +1,32 @@
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+import typescript from '@rollup/plugin-typescript';
+
+const production = !process.env.ROLLUP_WATCH;
+
+export default {
+	input: 'lib/index.ts',
```

Comment:
seems like 8 space tabs here, maybe we should use 2?

Reply:
It was actually an indent using a single tab. I've added eslint and fixed the problem.